python/lancedb/embeddings/__init__.py

```diff
@@ -12,6 +12,7 @@
 #  limitations under the License.
 
 
+from .cohere import CohereEmbeddingFunction
```

Comment:
is it worth separating out the existing embedding functions into their own modules? e.g., `openai.py`, `openclip.py`, `sentence_transformers.py`?

Reply:
yeah I think so. The scope and size of the file functions.py will become too large otherwise.